tests/libres_tests/res/enkf/test_active_list.py

```diff
@@ -62,3 +62,19 @@ def test_repr(self):
         self.assertTrue("PARTLY_ACTIVE" in rep)
         self.assertFalse("INACTIVE" in rep)
         self.assertFalse("ALL_ACTIVE" in rep)
+
+    def test_get_active_index_list(self):
```

Comment:
I think we need to add some more tests to this if we are to change it, partially to illustrate how to use `ActiveList´. And while we are at it the new tests might as well be from `pytest` instead of using `ResTest` as there is no benefit to using ´ResTest` here.

Reply:
Have added a new file with tests to be run with pytest: test_get_active_index_list.py under the tests directory

Reply:
If this is covered by the other test this can be removed.

Reply:
Have moved the test function from test_get_active_index_list.py to the existing one test_active_list.py and split it into several tests. Have renamed to get_active_index_list.